src/target/source/codegen_source_base.cc

```diff
@@ -113,5 +113,15 @@ void CodeGenSourceBase::EndScope(int scope_id) {
   indent_ -= 2;
 }
 
+std::string CodeGenSourceBase::SanitiseName(std::string name) {
```

Comment:
Can we use https://github.com/apache/tvm/blob/main/src/relay/backend/name_transforms.cc#L87-L95 ?

Reply:
Ah, I missed this one. Yes its better to use that. corrected